config/CLIConfiguration.ts

```diff
@@ -228,23 +237,53 @@ class _CLIConfiguration {
   }
 
   getDefaultAccount(): string | number | null {
-    return this.config && this.config.defaultAccount
-      ? this.config.defaultAccount
-      : null;
+    return (
+      this.getResolvedDefaultAccountForCWD() ||
+      this.config?.defaultAccount ||
+      null
+    );
+  }
+
+  getDefaultAccountOverrideFilePath(): string | null {
+    return findup([DEFAULT_ACCOUNT_OVERRIDE_FILE_NAME], {
+      cwd: getCwd(),
+    });
   }
 
-  // TODO a util that returns the account to use, respecting the values set in
-  // "defaultAccountOverrides"
-  // Example "defaultAccountOverrides":
-  //  - /src/brodgers/customer-project-1: customer-account1
-  //  - /src/brodgers/customer-project-2: customer-account2
-  // "/src/brodgers/customer-project-1" is the path to the project dir
-  // "customer-account1" is the name of the account to use as the default for the specified dir
-  // These defaults take precedence over the standard default account specified in the config
-  getResolvedDefaultAccountForCWD(
-    nameOrId: string | number
-  ): CLIAccount_NEW | null {
-    return this.getAccount(nameOrId);
+  getResolvedDefaultAccountForCWD(): number | null {
+    const defaultOverrideFile = this.getDefaultAccountOverrideFilePath();
+    if (!defaultOverrideFile) {
+      return null;
+    }
+    const source = fs.readFileSync(defaultOverrideFile, 'utf8');
+    const accountId = Number(source);
```

Comment:
What does `Number` spit out if you pass it something that's not a number string?

Reply:
It returns `NaN`--that's why I've included the `isNaN` check on line 261, so we can throw an error if the accountId cannot be converted to a number.